.github/ISSUE_TEMPLATE/4_onboarding_to_cluster.yaml

```diff
@@ -11,10 +11,10 @@ body:
           description: |
               Please select a cluster.
           options:
-              - Infra
-              - Morty
-              - Smaug
-              - Balrog
+              - moc/infra
```

Comment:
Is there any reason you changed it to be this way? We're implementing some automation to run this with jupyter notebooks and this would break this solution.

Reply:
I'm implementing a probot app to automate the onboarding process and wanted to hardcode as little in the controller as possible. To access files that need to be changed to onboard a project within the apps repo I needed to add moc or emea respectively.